agent/structs/connect_proxy_config.go

```diff
@@ -333,6 +333,9 @@ func (u *Upstream) Validate() error {
 	if u.DestinationName == "" {
 		return fmt.Errorf("upstream destination name cannot be empty")
 	}
+	if u.DestinationName == WildcardSpecifier && !u.CentrallyConfigured {
```

Comment:
So this can be wildcard when centrally configured?

Reply:
Yep, since the upstream defaults are stored until a wildcard name.